tests/Doctrine/Tests/Persistence/Mapping/SymfonyFileLocatorTest.php

```diff
@@ -84,6 +84,7 @@ public function testInvalidCustomNamespaceSeparator(): void
         $path   = __DIR__ . '/_files';
         $prefix = 'Foo';
 
+        /** @psalm-suppress NullArgument */
```

Comment:
Isn't this covered by the psalm config?

Reply:
It's covered in phpstan configuration. 

I could add it also to psalm config instead of this annotation here, I usually use psalm annotations because once it's fixed, psalm doesn't complain about ignoring an unexisting error, so I find it easy to remove it from here, but I'll move it if you want.

Reply:
Please move it, we like Psalm to complain about unused ignored rules, IMO it's less confusing for people who know something has been fixed in Psalm but still see the ignore rule.

Reply:
Done, what I meant is that Psalm does not complain about unused ignore rules.